cobigen-cli/.classpath

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This `.classpath` file should be added to the `.gitignore` list, so that it doesn't get pushed to the repo :)